scripts/templates/locals.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_organizations_organization" "root_account" {}
```

Comment:
data.aws_organization.root_account needed - can't see any use of it or the local vars that depend on it